fsh/ig-data/input/pages/workplace.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
```

Comment:
Did you create this file internationally?

Reply:
I'm not sure what you mean here, could you expand on your comment?

Reply:
You added this file, but why?

Reply:
Oh - I see! I thought I had removed it already, I was using it during the testing phase.

I will delete it.

Reply:
...it's now gone!